typescript-async-await/exceptions-best.ts

```diff
@@ -0,0 +1,46 @@
+import { read } from './read.js';
+
+// Keep track of app load time so each log message can be timed.
+// Log messages should all be approximately 1 second apart.
+const startTime = Date.now();
+const elapsed = (): string =>
+  `${Math.round((Date.now() - startTime) / 1000)}s -`;
+
+async function throwOnce(): Promise<void> {
```

Comment:
The intent of exceptions-best is to learn how exceptions can bubble up the call stack.

You should remove the try/catch from each of these functions and instead put a single try/catch around the calls to these functions, down at line 44. You will need to convert those from Promise syntax to `await` so you can wrap the calls in a try/catch.